test/functional/legacy/increment_spec.lua

```diff
@@ -720,4 +739,10 @@ describe('Ctrl-A/Ctrl-X on visual selections', function()
       eq({}, nvim.get_vvar('errors'))
     end)
   end
+
+  it('does not drop leading zeroes', function()
+    execute('set nrformats&vi') -- &vi makes Vim compatible
```

Comment:
Ohhh "original" as in "original vim", I get it. Thanks! Also thanks for the help link, I got it now. I'm gonna keep it this way though, to keep the changes kinda minimal.